src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonLink } from './Person';
+
+export const PeopleTable: React.FC = () => {
+  const [people, setIsPeoples] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  const getIsPeople = async () => {
+    try {
+      const loadePeople = await getPeople();
+
+      setIsPeoples(() => loadePeople);
+    } catch {
+      setError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const isLoaded = !isLoading && people.length;
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  useEffect(() => {
+    getIsPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+          {isLoaded && (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {people?.map(element => (
+                  <PersonLink
+                    person={element}
+                    people={people}
+                  />
+                ))}
+              </tbody>
+            </table>
+          )}
+
+          {error && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {(!isLoading) && (
```

Comment:
it's visible even when the table with people is displayed - it should be displayed only if fetched people array is empty

![image](https://github.com/mate-academy/react_people-table-basics/assets/86923089/f2414287-8bb2-47c6-ba20-d47b2c2b0ee4)